### PR TITLE
NAS-109867 / 21.04 / Chart release summary dialog styling issue

### PR DESCRIPTION
**Pull request by AlexKarpov98**

### CODE REVIEW

File: src/app/pages/applications/dialogs/chart-events/chart-events-dialog.component.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
-<div mat-dialog-content fxLayout="column wrap" fxLayoutAlign="center start" >
-    <div class="logo-container" fxLayout="row" fxLayoutAlign="center center">
+<div mat-dialog-content class="chart-events-dialog-content">
```
Comment (undsoft) [inline src/app/pages/applications/dialogs/chart-events/chart-events-dialog.component.html:1]: You don't seem to be using this class or am I missing something?

**Comment by AlexKarpov98** [comment]
@undsoft Fixed.

**Comment by AlexKarpov98** [comment]
@denysbutenko Fixed two issues you mentioned on slack. Please check them.

**Review by undsoft — APPROVED** [review]

**Review by denysbutenko — APPROVED** [review]
Looks good now 👍🏻

**Pull request merged**